example/HelloWorldFunction/src/main/java/helloworld/AppIdempotency.java

```diff
@@ -0,0 +1,85 @@
+package helloworld;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.lambda.powertools.idempotency.Idempotency;
+import software.amazon.lambda.powertools.idempotency.IdempotencyConfig;
+import software.amazon.lambda.powertools.idempotency.Idempotent;
+import software.amazon.lambda.powertools.idempotency.persistence.DynamoDBPersistenceStore;
+import software.amazon.lambda.powertools.utilities.JsonConfig;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AppIdempotency implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+    private final static Logger LOG = LogManager.getLogger();
+
+    public AppIdempotency() {
+        // we need to initialize idempotency configuration before the handleRequest method is called
+        Idempotency.config().withConfig(
```

Comment:
None of the test case seems to have a full example like this with a handler. Would it makes sense to add a test documenting config as well and have a handler example in the test `handler` package ?

Reply:
Tests are unit tests, and already cover 85% of the code. What would this test add? If this is for sample/documentation purpose, I don't think it is really useful. It will be documented and the sample will be available too.

Reply:
Yeah the intention was to have self documentation in the test case itself via lambda handler like we have for others too. If you don't find it useful thats ok, as long as the functionality is tested via unit test already. 


Its good to have self documentation via test case itself.

Reply:
added a end to end test with a handler #cd71b5f